packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -608,6 +609,17 @@ const ruleset: any = {
     /// ARM rules without an RPC code
     ///
 
+    ProvisioningStateMustBeReadOnly: {
+      description: "This is a rule introduced to validate if provisioningState property is set to readOnly or not.",
+      message: "{{error}}",
+      severity: "error",
+      resolved: true,
+      formats: [oas2],
+      given: ["$[paths,'x-ms-paths'].*.*.responses[*].schema"],
```

Comment:
I think this is equivalent:
```suggestion
      given: ["$[paths,'x-ms-paths'].*.*.responses.*.schema"],
```

Reply:
yeah & it'll be consistent as well!